### PR TITLE
add instructions and console binary

### PR DESCRIPTION
**Pull request by Ben-Haslam**

### CODE REVIEW

File: autonity-instructions.md [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+        --pprof \
+        --pprof.addr 0.0.0.0 \
+        --networkid  "65010000"\
+        --nousb \
+        --verbosity 3 \
+        --datadir ./autonity-chaindata \
+        --syncmode snap \
+        --nat extip:$(echo $IP_ADDRESS)
+
+    ```
```
Comment (Klazomenai) [inline autonity-instructions.md:99]: Remove extra tab across all the lines, and remove blank line 98.

File: autonity-instructions.md [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+$ autonity version
+Autonity
+Version: 0.8.4-internal
+Architecture: amd64
+Protocol Versions: [66]
+Go Version: go1.18
+Operating System: linux
+GOPATH=
+GOROOT=/usr/local/go
+
```
Comment (Klazomenai) [inline autonity-instructions.md:45]: Remove blank line in comment

File: autonity-instructions.md [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+  - gas (the amount of gas you are providing for the transaction)
+  - gas price (the amount of gas you are willing to pay for computing the transaction)
+
+### Setup
+
+1. Navigate to your Autonity NodeJS Console install directory  and initialise a console session, specifying the IP address of the node you will connect to. The connection is made over WebSockets to port 8546:
+
+    ```bash
+    ./console ws://<IP-ADDRESS>:8546
+    ```
```
Comment (Klazomenai) [inline autonity-instructions.md:214]: Remove tab

File: autonity-instructions.md [hunk 4]
```diff
@@ -0,0 +1,429 @@
...
+
+2. Enter the following in the NodeJS Console, specifying the private key of the account submitting the transaction:
+
+    ```javascript
+    const privatekey = '<PRIVATE_KEY>';
+    const account = web3.eth.accounts.wallet.add(privatekey);
+    const myAddress = web3.utils.toChecksumAddress(account.address);
+    const gas = 10000000;
+    const gasPrice = 10000000000;
+    ```
```
Comment (Klazomenai) [inline autonity-instructions.md:225]: Remove tab

File: autonity-instructions.md [hunk 5]
```diff
@@ -0,0 +1,429 @@
...
+await autonity.approve('<spender>', <amount>).send({from: myAddress, gas: gas, gasPrice: gasPrice})
+```
+
+#### Transfer Newton stake tokens as an approved account:
+
+```javascript
+await autonity.transferFrom('<sender>', '<recipient>', <amount>).send({from: myAddress, gas: gas, gasPrice: gasPrice})
+```
+
+
```
Comment (Klazomenai) [inline autonity-instructions.md:254]: Remove blank line

**Comment by Klazomenai** [comment]
@Ben-Haslam Should the file `console` to `console.sh` as its a shell script?

**Comment by Ben-Haslam** [comment]
> @Ben-Haslam Should the file `console` be renamed to `console.sh` as its a shell script?

no - its the nodejsconsole binary

**Comment by Klazomenai** [comment]
> > @Ben-Haslam Should the file `console` be renamed to `console.sh` as its a shell script?
> 
> no - its the nodejsconsole binary

Line 1: `#!/bin/sh`, making it technically a shell script, but hey.

**Review by Klazomenai — APPROVED** [review]
LGTM

**Pull request merged**